src/components/Heading/index.js

```diff
@@ -2,8 +2,8 @@ import classNames from 'classnames';
 
 import style from './style.scss';
 
-const Heading = ({ className, children }) => {
-  const headingClass = classNames(className, style.heading);
+const Heading = ({ className, children, color }) => {
```

Comment:
is this color property used anywhere?

Reply:
It's gonna be used on different View so I already prepared component for that and usin' it on different branch with new View.